packages/frontend/utils/useWindowSize.tsx

```diff
@@ -1,26 +1,30 @@
 import { useEffect, useState } from 'react';
 
 const mobileBreakpoint = 768;
+const tabletBreakpoint = 1024;
 
 interface WindowSize {
   width: number | undefined;
   height: number | undefined;
-  isMobile: boolean | undefined;
+  isMobile: boolean;
```

Comment:
Is `isMobile` still in use? Saw you removed/renamed earlier

Reply:
Not in use now, but it's referenced as a breaking point in Figma, so might need it.